include/cocaine/traits/tuple.hpp

```diff
@@ -324,6 +324,11 @@ struct type_traits<std::tuple<Args...>> {
     }
 };
 
+// Pair serialization
+
+template<typename T, typename U>
+struct type_traits<std::pair<T, U>> : type_traits<std::tuple<T, U>> {};
```

Comment:
My motivation is that people usually think that no specifier here means `private` because everybody is focused on classes, so this line might confuse them.